runtime/doc/news.txt

```diff
@@ -95,6 +95,15 @@ The following changes may require adaptations in user config or plugins.
   • Legacy and extmark signs are displayed and listed with the same priority:
       line number -> priority -> sign id -> recently placed
 
+==============================================================================
+BREAKING CHANGES IN HEAD                                    *news-breaking-dev*
```

Comment:
This shouldn't be here. `news.txt` is exclusively for changes between minor versions. This just becomes a rehash of the generated changelog anyway.